src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -6,25 +6,25 @@ import {
   ElementRef,
   Input,
   ChangeDetectorRef,
+  AfterViewInit,
 } from '@angular/core';
-import { LearningObjectService as PublicLearningObjectService } from 'app/cube/learning-object.service';
-import { OrderBy, Query, SortType } from 'app/interfaces/query';
-import { LearningObject, User } from '@entity';
+import { LearningObjectService as PublicLearningObjectService } from '../../../cube/learning-object.service';
+import { OrderBy, Query, SortType } from '../../../interfaces/query';
+import { LearningObject } from '../../../../entity';
```

Comment:
just an observation, we should add a [default import path](https://stackoverflow.com/a/49757060) in our `tsconfig` so we can just import by doing something like `~/entity` or `@/entity` instead of `../../../../entity`, etc..

Reply:
I disagree, the relative path is pretty safe to use. We've ran into issues before with default paths in our code not loading correctly based on bad build paths